### PR TITLE
[DPE-1770] Minor version upgrades

### PR DESCRIPTION
**Pull request by dragomirp**
Minor version upgrades.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -135,9 +136,9 @@ jobs:
             echo "mark_expression=" >> $GITHUB_OUTPUT
           else
             echo Skipping unstable tests
-            echo "mark_expression=not unstable" >> $GITHUB_OUTPUT
+            echo "mark_expression=and not unstable" >> $GITHUB_OUTPUT
```
Comment (dragomirp) [inline .github/workflows/ci.yaml:139]: This should fix the scheduled tests.

File: src/charm.py [hunk 2]
```diff
@@ -169,6 +148,39 @@ def _on_install(self, _) -> None:
...
+    def _on_install(self, _) -> None:
+        """On install hook.
+
+        This initialises local config files necessary for pgbouncer to run.
+        """
+        self.unit.status = MaintenanceStatus("Installing and configuring PgBouncer")
+
+        # Install the charmed PostgreSQL snap.
+        try:
+            self._install_snap_packages(packages=SNAP_PACKAGES)
```
Comment (taurus-forever) [inline src/charm.py:160]: should we hold the snap?

**Review by taurus-forever — APPROVED** [review]
Nice job, just a one question otherwise LGTM!

File: src/charm.py (hunk 2 above)
Comment (dragomirp) [reply src/charm.py:160]: If it's installed by revision, it should be held in [`_install_snap_packages()`](https://github.com/canonical/pgbouncer-operator/blob/f331c218907afef40909df1837e5edd19df75654/src/charm.py#L665)

**Review by marceloneppel — APPROVED** [review]

**Pull request merged**